docs/architecture/cloud-native/elastic-search-in-azure.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Elasticsearch in cloud-native applications
+description: Learn about adding Elastic Search capabilities to cloud-native applications.
+author: robvet
+ms.date: 12/28/2019
+---
+# Elasticsearch in a cloud-native app
+
+[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+
+Elasticsearch is a distributed search and analytics system that enables complex search capabilities across diverse types of data. It's open source and widely popular. Consider how the following companies integrate Elasticsearch into their application:
+
+- [Wikipedia](https://blog.wikimedia.org/2014/01/06/wikimedia-moving-to-elasticsearch/) for full-text and incremental (search as you type) searching.
+- [GitHub](https://www.elastic.co/customers/github) to index and expose over 8 million code repositories.  
+- [Docker](https://www.elastic.co/customers/docker) for making its container library discoverable.
+
+Elasticsearch is built on top of the [Apache Lucene](https://lucene.apache.org/core/) full-text search engine. Lucene provides high-performance document indexing and querying. It indexes data with an [inverted indexing](https://www.baeldung.com/lucene) scheme – instead of mapping pages to keywords, it maps keywords to pages just like a glossary at the end of a book. Lucene has powerful [query syntax](https://www.baeldung.com/lucene)  capabilities and can query data by:
```

Comment:
```suggestion
Elasticsearch is built on top of the [Apache Lucene](https://lucene.apache.org/core/) full-text search engine. Lucene provides high-performance document indexing and querying. It indexes data with an [inverted indexing](https://www.baeldung.com/lucene) scheme – instead of mapping pages to keywords, it maps keywords to pages just like a glossary at the end of a book. Lucene has powerful [query syntax](https://www.baeldung.com/lucene) capabilities and can query data by:
```